### DISCUSSION TITLE
Including pages on other pages possible?

### DISCUSSION THREAD

**Discussion by Verequus** [requarks/wiki#4870] [General]
I'd like to mark some parts of a page so it is includeable on another page. For example, I have a table that should be displayed at two different places, but I don't want to duplicate the table itself to avoid that the two tables end up out of sync. Is something like that possible? Or can this functionality be added, if this is missing?

**Comment by NGPixel** [comment]
This is not possible at the moment, but is planned for 3.0.

**Reply by Verequus** [reply]
Is there already a release date known already for this version?

**Reply by plangendoen** [reply]
When will this feature be available? When is 3.0 planned to be released?
